TouchVisualizer/Visualizer.swift

```diff
@@ -40,7 +40,7 @@ final public class Visualizer:NSObject {
     
     // MARK: - Helper Functions
     @objc internal func applicationDidBecomeActiveNotification(_ notification: Notification) {
-        UIApplication.shared.keyWindow?.swizzle()
+        UIApplication.shared.swizzle()
```

Comment:
Wait from our conversation during Daily: I thought we couldn't access `UIApplication.shared` instance here.

Reply:
Ah it was a mistake,
I was working with an example app and I had removed this method applicationDidBecomeActiveNotification. 
Denis reminded me about using the shared instance here 👍